users/views.py

```diff
@@ -264,6 +264,15 @@ def get(self, request):
         return HttpResponseRedirect(reverse("users:me"))
 
 
+class Tutorial(TemplateView):
+    template_name = "tutorial.html"
+
+class TutorialDone(LoginRequiredMixin, View):
+    def get(self, request, *args, **kwargs):
```

Comment:
While I can see why you did this, the http pedant in me says this really aught to be a POST method, especially as this is technically susceptible to a (very minor) CSRF attack

Reply:
Deciding to leave this unfixed - the effort in getting the CSRF token in an AJAX request seems pretty major, to the point where I don't trust myself to correctly add a script that fixes this _to every page on the website_. (Especially as it seems like this involves Django having to set a lot more CSRF cookies.)